raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java

```diff
@@ -446,10 +446,16 @@ private void insertRaster(String fileName, String absPath, Boolean rasterExists)
 	public static String getPropertiesPath(String rasterDir) throws IOException {
 
 		File propsRootDir = new File("/opt/geoserver_data/coverages/");  // root dir of geoserver imgMosaic. Must be mapped as volume by container!
-		File rasterFile = new File(rasterDir); // extract path from downloaded raster file
-		String rasterRootDir = rasterFile.getAbsoluteFile().getParent();
 		String propPath = new String(); // Path to specific mosaic to be found 
+		
+		if (!propsRootDir.exists()) { 		// if this is the first raster ever received, the GeoServer coverages folder might be uninitiated
+			propPath = "0";

```

Comment:
No need for `"0"` here. Returning `null` would be sufficient her IMHO.

Reply:
The propPath is returned to [here](https://github.com/meggsimum/sauber-sdi-docker/blob/bb479d80bcbe77c4c91489ee37414fee1a481059/raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java#L480) and will be inserted as `null` into the `station_data.raster_metadata` table [here](https://github.com/meggsimum/sauber-sdi-docker/blob/bb479d80bcbe77c4c91489ee37414fee1a481059/raster_download/src/main/java/de/meggsimum/sauber/sdi/RasterDownloader.java#L485).

Would this not insert `null` on the dir not existing, and `0` on the dir existing but no relevant CovStore? If so I'm happy to change this if this is intended.